utils/docker/Dockerfile.leap.15

```diff
@@ -1,4 +1,4 @@
-# Copyright (C) 2018-2020 Intel Corporation
+# Copyright (C) 2018-2021 Intel Corporation
```

Comment:
I can't comment...what is the empty file above? utils/docker/Dockerfile.code_scanning

Reply:
Permissions change.  There were two files, but I didn't do anything with the contents of that one.